packages/Frontend/src/components/Navigation/SidebarLeftOverlay.test.jsx

```diff
@@ -91,14 +92,12 @@ describe('SidebarLeftOverlay', () => {
       expect(typeof inst.handleLogout).toBe('function');
     });
 
-    it('should call props.logout', () => {
-      const logout = jest.fn();
+    it('should call firebase.auth().logout', () => {
```

Comment:
I changed this test to say that it doesn't call the logout prop that we were passing in, but that it does call auth().logout like you saw in the SidebarLeftOverlay file.